src/Microsoft.AspNetCore.SignalR.Client/HubConnectionFactory.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.SignalR.Internal.Protocol;
+using Microsoft.AspNetCore.Sockets;
+using Microsoft.AspNetCore.Sockets.Client;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.SignalR.Client
+{
+    public static class HubConnectionFactory
+    {
+        public static HubConnection Create(Uri url, ILoggerFactory loggerFactory = null)
+        {
+            var httpConnection = new HttpConnection(url, loggerFactory);
```

Comment:
Hmm, should we name the factory such that it implies that the underlying connection is an `HttpConnection`? Since it can only create one flavor of `HubConnections`

Reply:
Maybe. I don't think people will care for now if there is no server that can handle this.